src/main/proto/th2_grpc_common/common.proto

```diff
@@ -163,13 +167,18 @@ message ValueFilter {
     string simple_filter = 3;
     MessageFilter message_filter = 4;
     ListValueFilter list_filter = 5;
+    ComplexFilter  complex_filter= 6;
```

Comment:
I think we should call it SimpleList or something like that. It sounds more correctly in my oppinion